database/rrdhost.c

```diff
@@ -709,6 +709,19 @@ void rrdhost_save_charts(RRDHOST *host) {
     rrdhost_unlock(host);
 }
 
+static int is_valid_label_value(char *key) {
```

Comment:
Why do you use a `key` variable if it is a value?

Reply:
I copied and paste from a function, because I had to do only small changes, but I agree that to rename the pointer will let the code more readable.

Reply:
I copied and paste from a function, because I had to do only small changes, but I agree that to rename the pointer will let the code more readable.